composer.json

```diff
@@ -39,7 +39,7 @@
         "symfony/validator": "^4.4|^5.0",
         "symfony/yaml": "^4.4|^5.0",
         "willdurand/hateoas-bundle": "^2.0",
-        "winzou/state-machine-bundle": "^0.3|^0.4"
+        "winzou/state-machine-bundle": "^0.4.3"
```

Comment:
Why is it needed?

Reply:
There's a bug on winzou with dependencies between state-machine and state-machine bundle. Maybe there's something better to do:
"winzou/state-machine-bundle": "^0.3.2|^0.4.3"
I'm gonna try that.

Reply:
@lchrusciel But I think we have to improve CI configuration to test these two state machine versions.